legend-engine-xt-serviceStore-executionPlan/pom.xml

```diff
@@ -102,6 +102,28 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-xt-serviceStore-protocol</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-authentication-protocol</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
why depend on example module?

Reply:
It has the CredentialProvider class/api(s)

Reply:
Then we should move it. Depending prod flow on example seems wierd

Reply:
Its just the name of the module we need to change. Ephrim did n't have a good idea of what exactly to name this new module when he first pushed it. Maybe legend-engine-xt-authentication-connection-core ? 
or we can actually delete the example module and move everything it has to legend-engine-xt-authentication-implementation-core. Not sure if there is any advantage of keeping two modules.